content/english/episodes/ep-8.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Episode 8: with Mojdeh Rastgoo"
+meta_title: "Episode 8 with Mojdeh Rastgoo"
+description: "Learn about Mojdeh's journey. Since discovering Python and open source community in 2016, Mojdeh has been actively contributing to the community and striving for diversity and inclusion by pariticapting in DEI, CoC communities and co-organzing meetups"
+date: 2025-04-15T05:00:00-07:00
+host: ["Tereza", "Georgi"]
+draft: false
+tags: ["python", "iran", "france", "pyladies", "psf-coc-wg", "pypodcats", "community", "project manager"]
+publish_date: 2025-04-15T00:00:00-07:00
+podcast_file: "/audio/hidden-figures-ep8.mp3"
+podcast_duration: 1647.6
+episode_image: "images/ep8-mojdeh.png"
+podcast_bytes: "66615526" # the length of the episode in bytes
+images: ["images/ep8-mojdeh.png", "images/mojdeh.jpg"]
+explicit: false 
+type: 'episode'
+episode: '8'
+season: 1
+transcript: '/transcript/hidden-figures-of-python-ep8.srt'
+episode_type: full
+layout: episode
+youtube_id: "Gh1OOv2qtTA"
+  
+speaker:
+  url: "/speakers/mojdeh-rastgoo/"
+  title: "Mojdeh Rastgoo"
+  image: "/images/mojdeh.jpg"
+  description: "
+Mojdeh Rastgoo is a Machine Learning Data Scientist and Project Manager based in Pris.
+She has a PhD in computer sicence and has been an active memnber of the community since 2018. 
+She is a co organizer of PyLadies Paris and member of PSF Code of Conduct Working-Group since mid 2023.
+And recently she has joined the PyPodcats team as the fifth organizer.  
+
+She participated in organization of EuroSciPy 2023/2024 conference, JupyterCon 2023, PyCon.DE & PyData Berlin 2024, and PyLadies Paris meetups since 2022. 
```

Comment:
```suggestion
She participated in organization of EuroSciPy 2023/2024 conference, JupyterCon 2023, PyCon DE & PyData Berlin 2024, and PyLadies Paris meetups since 2022. 
```